frigate/embeddings/functions/onnx.py

```diff
@@ -184,7 +191,8 @@ def __call__(
                     onnx_inputs[key].append(value[0])
 
         for key in onnx_inputs.keys():
-            onnx_inputs[key] = np.array(onnx_inputs[key])
+            if onnx_inputs[key]:
```

Comment:
this would need to be a `.get()` right? otherwise it will fail on keyerror if it is not in there

Reply:
sure, that can be more robust